### ISSUE TITLE
Some third party lib can not fetch in China since the GFC

### ISSUE THREAD

**Issue by jinfagang** [microsoft/onnxruntime#1775]
```
fatal: unable to access 'https://chromium.googlesource.com/chromium/llvm-project/llvm/lib/Fuzzer/': Failed to connect to chromium.googlesource.com port 443: Connection timed out
fatal: clone of 'https://chromium.googlesource.com/chromium/llvm-project/llvm/lib/Fuzzer' into submodule path '/media/jintian/netac/ai/libs/onnxruntime/cmake/external/grpc/third_party/bloaty/third_party/libFuzzer' failed
Failed to clone 'third_party/libFuzzer'. Retry scheduled
Cloning into '/media/jintian/netac/ai/libs/onnxruntime/cmake/external/grpc/third_party/bloaty/third_party/libFuzzer'...
fatal: unable to access 'https://chromium.googlesource.com/chromium/llvm-project/llvm/lib/Fuzzer/': Failed to connect to chromium.googlesource.com port 443: Connection timed out
fatal: clone of 'https://chromium.googlesource.com/chromium/llvm-project/llvm/lib/Fuzzer' into submodule path '/media/jintian/netac/ai/libs/onnxruntime/cmake/external/grpc/third_party/bloaty/third_party/libFuzzer' failed
Failed to clone 'third_party/libFuzzer' a second time, aborting
Failed to recurse into submodule path 'cmake/external/grpc/third_party/bloaty'
Failed to recurse into submodule path 'cmake/external/grpc'
```

These git url can not  access behind China wall. any idea to reduce it?

**Comment by faxu** [comment]
this is the same issue you filed previously, right? https://github.com/microsoft/onnxruntime/issues/1601 

it's still on the backlog, so if this is blocking some of your projects, is this something you could contribute?

**Comment by jinfagang** [comment]
The only thing I can contribute is make libFuzzer local since this lib has only on Readme.txt inside... no codes..

Also it's not a third lib of onnxruntime, it's a third lib of grpc, and not a third lib of grpc directly, it's a third lib of bloaty which is third lib of grpc which is third lib of onnxruntime.

So .........

I think should make this annoying lib `cmake/external/grpc/third_party/bloaty`  gone....

**Comment by maidouxiaozi** [comment]
How to operate specifically？

**Comment by smilexin** [comment]
So, how to just skip this third party sync step?

**Comment by sicccs** [comment]
挂代理多更新几次子模块
git submodule update --init --recursive

**Comment by smilexin** [comment]
@sicccs not work

**Comment by smilexin** [comment]
after sync libFuzzer error, use following flag to continue build, which can skip git submodule update: 
--skip_submodule_sync

**Comment by snnn** [comment]
resolved in #2744
